tests/cnf/ran/talm/tests/talm-blockingcr.go

```diff
@@ -137,8 +137,8 @@ var _ = Describe("TALM Blocking CRs Tests", Label(tsparams.LabelBlockingCRTestCa
 	})
 
 	When("a blocking CR is missing", func() {
-		// 47948 - Tests multiple UOCRs can be enabled in parallel with blocking CR.
-		It("verifies CGU is blocked until blocking CR created and succeeded", reportxml.ID("47948"), func() {
+		// 47956 - Tests multiple UOCRs can be enabled in parallel with missing blocking CR.
```

Comment:
@josclark42 you can consider updating polarion test description to match the current design? e.g., UOCR > CGU. can be in separate PR as there are probably other test cases need a better name.

Reply:
@yliu127 This is an easy, necessary change so I updated the four occurances of UOCR to CGU. I'll update Polarion as well.